astromon/scripts/excluded_region.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+"""
+Script to add excluded regions to astromon.db
+"""
+
+import argparse
+import getpass
+import logging
+
+from astropy.coordinates import SkyCoord
+from ska_helpers.logging import basic_logger
+
+from astromon import db
+
+logger = basic_logger("regions", level="INFO")
+
+
+class ArgumentError(Exception):
+    pass
+
+
+def get_parser():
+    config_levels = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+    config_levels += [x.lower() for x in config_levels]
+
+    parser = argparse.ArgumentParser(
+        description="Add excluded regions to astromon.db",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "action",
+        choices=list(ACTIONS),
+        help="Action to perform",
+    )
+    parser.add_argument(
+        "--ra",
+        help="Right Ascension of the center of the excluded region (in degrees)",
+    )
+    parser.add_argument(
+        "--dec",
+        help="Declination of the center of the excluded region (in degrees)",
+    )
+    parser.add_argument(
```

Comment:
Do we need a --dry-run mode?

Reply:
Though I suppose since the region script doesn't do any _checks_ (like if the region already exists etc) that it doesn't really matter.  And one could use remove_region to just remove a typo region if noticed.

Reply:
yes, what would we do with a "dry-run" mode? What is it that we are checking?